main/src/ca/uwaterloo/flix/api/lsp/provider/CompletionProvider.scala

```diff
@@ -87,9 +87,18 @@ object CompletionProvider {
       case Some(context) =>
         root match {
           case Some(nonOptionRoot) =>
-            getCompletions()(context, flix, index, nonOptionRoot, deltaContext) ++
-            (FromErrorsCompleter.getCompletions(context)(flix, index, nonOptionRoot, deltaContext)
-              map (comp => comp.toCompletionItem(context)))
+            val comp = getCompletions()(context, flix, index, nonOptionRoot, deltaContext) ++
+            FromErrorsCompleter.getCompletions(context)(flix, index, nonOptionRoot, deltaContext)
+            CompletionRanker.findBest(comp, deltaContext) match {
+              case None => comp map(comp => comp.toCompletionItem(context))
+              case Some(completionForBoost) =>
+                val elemForBoost = completionForBoost.toCompletionItem(context)
+                val boostedElem = CompletionItem(elemForBoost.label, "1" + elemForBoost.sortText.splitAt(1)._2, elemForBoost.filterText,
```

Comment:
This is currently a hack to boost the completion to the top of the popup pane. I don't know if we have to change all the types of `Completion` to hold `sortText` and that way change the priority..